### ISSUE TITLE
Frequently requires me to "reopen"

### ISSUE THREAD

**Issue by KileyAdams92** [Microsoft/vscode#52730]
Issue Type: <b>Bug</b>

Terminal and files will freeze, then I am prompted to reopen or close visual studio code. This happens frequently (3 times in 10 minutes). Usually when I have my application started and resave a file. 

VS Code version: Code 1.24.1 (24f62626b222e9a8313213fb64b10d741a326288, 2018-06-13T17:51:32.889Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700HQ CPU @ 2.60GHz (8 x 2592)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: disabled_software<br>video_decode: enabled<br>video_encode: enabled<br>vpx_decode: unavailable_software<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.87GB (8.85GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code\Code.exe|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (5)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|1.4.12
prettier-vscode|esb|1.5.0
dotenv|mik|1.0.1
html5-boilerplate|sid|1.0.3
vscode-babel-repl|t-s|0.1.2

</details>

**Comment by chrmarti** [comment]
What do you get for `code --status` when this happens? Also check https://github.com/Microsoft/vscode/wiki/Performance-Issues for more.

**Comment by KileyAdams92** [comment]
Do you mean what does it say during the time it freezes if I do a code --status or after? During I can't type anything in the terminal.

**Comment by KileyAdams92** [comment]
![cap2](https://user-images.githubusercontent.com/31576550/41875776-919de574-7891-11e8-9870-b7563278fc2e.PNG)
![cap3](https://user-images.githubusercontent.com/31576550/41875777-91afb8a8-7891-11e8-859c-364e964ecd1e.PNG)
![cap4](https://user-images.githubusercontent.com/31576550/41875778-91c7a166-7891-11e8-84a0-8641e57399ce.PNG)
![capture](https://user-images.githubusercontent.com/31576550/41875779-91deec9a-7891-11e8-9f6d-06fb42e717e8.PNG)


I'm not able to type during the issue, however this is what my "code --status" shows afterwards and also a few of the pop-ups that show during the error.

**Comment by KileyAdams92** [comment]
![image](https://user-images.githubusercontent.com/31576550/41876272-e4f654b2-7892-11e8-914f-2dc352fe95af.png)

**Comment by jrieken** [comment]
cc @Tyriar for the terminal. can the gpu-renderer freeze the window?

**Comment by jrieken** [comment]
@KileyAdams92 When the freeze happens can you follow the second part of this section: https://github.com/Microsoft/vscode/wiki/Performance-Issues#the-rendererwindow-process-consumes-a-lot-of-cpu. Mainly, creating a CPU profile using dev tools? That will precise point to the code that going crazy

**Comment by Tyriar** [comment]
I've heard this once before, were you able to use the DOM-based renderer as suggested in the notification?

**Comment by KileyAdams92** [comment]
I'm going to be completely honest, I'm not sure how to do that.

**Comment by mark-adams** [comment]
It looks like this might be the same as #51490.
Can you try going into your preferences and set `terminal.integrated.rendererType` to `dom`?

**Comment by Tyriar** [comment]
@mark-adams yep that's it. @KileyAdams92 that notification will do it for you by hitting the yes button, if it gave you a chance that is.
